app/src/i18n/index.ts

```diff
@@ -18,7 +18,7 @@ const initI18next = async (lng: string, ns: string) => {
   return i18nInstance;
 };
 
-export async function useTranslation(
+export async function translationFunc(
```

Comment:
Isn't this used in a bunch of places? Why do we have to rename it? It is a hook after all.

Reply:
I'll fix this